tutorials/01-Command-Line/Challenge 1/Text Files/generate.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
```

Comment:
Please ensure your code runs under `#!/bin/sh` since MacOS has [deprecated bash](https://www.theverge.com/2019/6/4/18651872/apple-macos-catalina-zsh-bash-shell-replacement-features).

Reply:
Resolved in https://github.com/madetech/learn/pull/165/commits/e0ff53f07eb4edbfee6d84c04bad27521816de7e